### PR TITLE
WIP[feat]: Wishlist

### PR DESCRIPTION
**Pull request by saadahmadhere**
image
image

### CODE REVIEW

**Review by rahul0396 — APPROVED** [review]
Code optimized and clean. Good to be Merged

File: css/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+main {
+  margin: var(--size-8) auto;
+}
+
+main .h1 {
```
Comment (MaruthiPothuganti) [inline css/wishlist.css:11]: Avoid Styling with Tag names.

File: css/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+.logo {
+  margin-right: var(--size-2);
+}
+
+.btn_logout {
+  padding: var(--size-3) var(--size-6);
+}
+
+@media only screen and (min-width: 30rem) {
```
Comment (MaruthiPothuganti) [inline css/wishlist.css:33]: Its a good practice to place media queries at the end of CSS file

**Review by MaruthiPothuganti — COMMENTED** [review]
Suggested some changes. Everything lelse looks good. Made pretty good use of utility classes.

File: css/wishlist.css (hunk 2 above)
Comment (saadahmadhere) [reply css/wishlist.css:33]: Thanks for pointing that out. Done!

File: css/wishlist.css (hunk 1 above)
Comment (saadahmadhere) [reply css/wishlist.css:11]: Done!

**Pull request merged**